_posts/2015-07-25-bear-foraging.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: It's hot out there for a bear
+author: neuroamanda
+---
+
+_Foraging for food takes precedence over staying cool_
+
+Grizzly bears have a dilemma.
+They only have about seven months a year to eat 12 months' worth of food before they go into hibernation.
```

Comment:
Not clear: you don't need to say "before they go into hibernation"; it might be better to say "since they hibernate for the remaining five months"?